pkg/util/log/slog_adapter.go

```diff
@@ -0,0 +1,30 @@
+package log
+
+import (
+	"log/slog"
+
+	"github.com/go-kit/log"
+	sloggk "github.com/tjhop/slog-gokit"
+)
+
+// GoKitLogToSlog convert go-kit/log to slog
+// usage: logutil.GoKitLogToSlog(gokitLogger)
+func GoKitLogToSlog(logger log.Logger) *slog.Logger {
```

Comment:
I wonder if we should name this function to `PrometheusLoggerToSlot` and have another function for `GoKitLogToSlog` which takes an additional level parameter?